.github/workflows/release.yml

```
@@ -35,3 +35,10 @@ jobs:
           tag_name: ${{ github.event.inputs.tag }}
           generate_release_notes: true
           target_commitish: ${{ github.sha }}
+
+  gradle_build_and_upload:
```

Nit, if you think that makes sense, to keep the same formatting:
```suggestion
  gradle-release:
```